website/content/configuration/_index.md

```diff
@@ -4,28 +4,53 @@ weight: 4
 ---
 
 MetalLB remains idle until configured. This is accomplished by
-creating and deploying a config map into the same namespace
+creating and deploying various resources into the same namespace
 (metallb-system) as the deployment.
 
-There is an example config map in
-[`config/samples/example-config.yaml`](https://raw.githubusercontent.com/metallb/metallb/main/config/samples/example-config.yaml),
+There are various examples of the configuration CRs in
+[`config/samples`](https://raw.githubusercontent.com/metallb/metallb/main/config/samples),
 annotated with explanatory comments.
 
-If you've named the config map `config.yaml`, you can deploy the manifest with `kubectl apply -f config.yaml`.
+Also, the API is [fully documented here](../apis/_index.md).
 
-{{% notice note %}}
-If you installed MetalLB with Helm, you will need to change the
-namespace of the config map to match the namespace in which MetalLB was
-deployed, and change the name of the config map from `config` to
-`metallb`.
-{{% /notice %}}
+## Defining the IPs to assign to the Load Balancer services
+
+In order to assign an IP to the services, MetalLB must be instructed to do so via the
+`IPAddressPool` CR.
+
+All the IPs allocated via `IPAddressPool`s contribute to the pool of IPs that MetalLB
+uses to assign IPs to services.
+
+```yaml
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: first-pool
+  namespace: metallb-system
+spec:
+  addresses:
+  - 192.168.10.0/24
+  - 192.168.9.1-192.168.9.5
+  - fc00:f853:0ccd:e799::/124
+```
+
+Multiple instances of `IPAddressPool` can co-exist, pools can defined by CIDR, by range,
+and both IPV4 and IPV6 address can be assigned.
+
+## Announce the service IPs
+
+Once the IPs are assigned to a service, they must be announced.
 
 The specific configuration depends on the protocol(s) you want to use
 to announce service IPs. Jump to:
 
 - [Layer 2 configuration](#layer-2-configuration)
 - [BGP configuration](#bgp-configuration)
-- [Advanced configuration](#advanced-address-pool-configuration)
+- [Advanced BGP configuration](./_advanced_bgp_configuration.md)
+- [Advanced L2 configuration](./_advanced_bgp_configuration.md)
+- [Advanced IPAddressPool configuration](./_advanced_bgp_configuration.md)
```

Comment:
should be `./_advanced_l2_configuration.md` and `./_advanced_ipaddresspool_configuration.md`